src/components/file-viewer/examples/file-viewer.tsx

```diff
@@ -39,6 +39,20 @@ export class FileViewerExample {
             <limel-file-viewer url="https://file-examples-com.github.io/uploads/2017/10/file_example_ODS_5000.ods" />,
             <h5>Presentation `.odp`</h5>,
             <limel-file-viewer url="https://file-examples-com.github.io/uploads/2017/10/file_example_ODP_1MB.odp" />,
+            <h5>Odp file `.odp`</h5>,
+            <limel-file-viewer url="https://filesamples.com/samples/document/odp/sample3.odp" />,
+            <h5>Microsoft word `.dotx`</h5>,
+            <limel-file-viewer url="https://example-files.online-convert.com/document/dotx/example.dotx" />,
+            <h5>Presentation `.pptx`</h5>,
+            <limel-file-viewer url="https://scholar.harvard.edu/files/torman_personal/files/samplepptx.pptx" />,
+            <h5>Microsoft word `.doc`</h5>,
+            <limel-file-viewer url="ehttps://file-examples-com.github.io/uploads/2017/02/file-sample_1MB.doc" />,
+            <h5>Microsoft PowerPoint `.pps`</h5>,
+            <limel-file-viewer url="https://israeltekahun.com/pps.pps" />,
+            <h5>Microsoft PowerPoint `.ppsx`</h5>,
+            <limel-file-viewer url="https://israeltekahun.com/ppsx.ppsx" />,
+            <h5>Microsoft PowerPoint `.ppsm`</h5>,
+            <limel-file-viewer url="https://israeltekahun.com/ppsm.ppsm" />,
```

Comment:
By the way; i didn't mean we should add them here in the documentation. I meant we should test and make sure that what we expect to be supported by the online viewer is actually supported, and the Office viewer shows them :) 

However, we could add a list of supported extensions somewhere in the documentations.

Reply:
Exactly, these will not be added in the final merge. They are there to only test if the formats are supported or not, just for us to find out if that is the case.